tfa.module

```diff
@@ -27,6 +27,71 @@ function tfa_help($route_name, RouteMatchInterface $route_match) {
 }
 
 /**
+ * Get TFA data for an account.
+ *
+ * @param object $uid
+ *   User account.
+ *
+ * @return array
+ *   TFA data.
+ */
+function tfa_get_tfa_data($uid) {
```

Comment:
You should inspect each one of these callbacks and move them into 1. an entity, 2. a service, 3. a class